docs/how_to.md

```diff
@@ -395,6 +456,7 @@ x-terraform-force-new | boolean |  If the value of this property is updated; ter
 x-terraform-sensitive | boolean |  If this meta attribute is present in an object definition property, it will be considered sensitive as far as terraform is concerned, meaning that its value will not be disclosed in the TF state file
 x-terraform-id | boolean | If this meta attribute is present in an object definition property, the value will be used as the resource identifier when performing the read, update and delete API operations. The value will also be stored in the ID field of the local state file.
 x-terraform-field-name | string | This enables service providers to override the schema definition property name with a different one which will be the property name used in the terraform configuration file. This is mostly used to expose the internal property to a more user friendly name. If the extension is not present and the property name is not terraform compliant, an automatic conversion will be performed by the OpenAPI Terraform provider to make the name compliant (following Terraform's field name convention to be snake_case) 
+x-terraform-field-status | boolean | If this meta attribute is present in an object definition property, the value will be used as the status identifier when executing the polling mechanism on eligible async operations such as POST/PUT/DELETE.
```

Comment:
Is this really supposed to be a boolean?

Reply:
yep, also keep in mind that the status field must be computed hence the property will not be exposed in the terraform configuration so being able to change the status field name will not make any difference...this is more a flag that allows the provider to specify which status field should be used in case the schema is lacking a field named 'status' or they preferred to use a different property as the status field.

makes sense?

Reply:
as discussed, a better design would be to move the property to a higher level and will create a card for that bit of work so we can make the change to both this extension as well as the id one at once.

Example of the enhancement proposed:
````
  LBV1:
    type: "object"
    x-terraform-id: "id"
    x-terraform-status-field: "status"
    required:
      - name
      - backends
    properties:
````